scripts/release/verify-fdroid-native-libs.sh

```
@@ -52,18 +186,59 @@ for apk in "${apks[@]}"; do
-  printf '  %s\n' "${failures[@]}" >&2
-  exit 1
-fi
+for apk in "${apks[@]}"; do
+  apk_name="$(basename "$apk")"
+  verify_dir="$tmp_dir/verify-$apk_name"
+  mkdir -p "$verify_dir"
+  unzip -qq "$apk" "lib/*/*.so" -d "$verify_dir"
+
+  while IFS= read -r so_file; do
+    if has_build_id "$so_file"; then
+      rel_path="${so_file#$verify_dir/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)